Ginger/GingerPluginCore/RecordingLib/ElementRecordEventArgs.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amdocs.Ginger.Plugin.Core
+{
+    public class ElementActionCongifuration
+    {
```

Comment:
Need to change class & file name from "ElementActionCongifuration" to "ElementRecordedEventArgs"

Reply:
it was already having the name as "ElementRecordedEventArgs" but since we require it in "WebPlatForm" so we decided it to be renamed as "ElementActionConfiguration", please let me know if we still need to rename it?

Reply:
i don't really like having those classes on GingerPluginCore but ok for now, so please change also file name to be "ElementActionCongifuration" because currently file name is diffrent from class name